src/binder/binder_context.cpp

```diff
@@ -94,7 +94,7 @@ bool BinderContext::GetColumnPosTuple(
     value_type = column_object->GetColumnType();
     return true;
   } catch (CatalogException &e) {
-    LOG_TRACE("Can't find table %d! Return false", std::get<1>(table_id_tuple));
+    LOG_TRACE("Can't find table %s! Return false", table_obj->GetTableName().c_str());
```

Comment:
Remove the trace line. 
As you can see the old line refers to finding a table, in the current code a table_obj is passed in (which you use). As far as this particular trace line is concerned, it isn't useful any more.

Reply:
Ah, good point. I revisited it and not even the trace this try, catch block is superfluous too since there's nothing in that code anymore that throws CatalogException.